### PR TITLE
feat: add k8s namespace indicator

### PR DESCRIPTION
**Pull request by ofirshtrull**
## Description
I needed to know on which NS i was at all time because I have many contexts and namespaces under my supervision 

## Motivation and Context
mistakes can always happen and I want to limit them where I can :-)

## How Has This Been Tested?
I tested this on my terminal :-)

## Screenshots (if appropriate):
Screen Shot 2021-06-03 at 17 56 47

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

File: docs/themes-list/powerline-base.rst [hunk 1]
```diff
@@ -75,6 +75,7 @@ The contents of the prompt can be "reordered", all the "segments" (every piece o
 * ``in_toolbox`` - Show identifier if running inside a `toolbox <https://github.com/containers/toolbox>`_
 * ``in_vim`` - Show identifier if running in ``:terminal`` from vim
 * ``k8s_context`` - Show current kubernetes context
+* ``k8s_namespace_prompt`` - Show current kubernetes namespace
```
Comment (davidpfarrell) [inline docs/themes-list/powerline-base.rst:78]: I think you just want `k8s_namespace` here, unless the actual prompt function is named `__powerline_k8s_namespace_prompt_prompt` ? :)

**Review by davidpfarrell — CHANGES_REQUESTED** [review]
Greetings!

One small comment for the RST file, but otherwise this looks like a clean, useful addition.

**Comment by ofirshtrull** [comment]
Noooooo so close to a perfect pr 😅
Will fix shortly

Thanks fir spotting the mistake

**Comment by ofirshtrull** [comment]
@davidpfarrell i fixed it

**Review by davidpfarrell — APPROVED** [review]
Looks good, thanks for contributing !

**Comment by ofirshtrull** [comment]
happy to share things i add for me

**Pull request merged**